client/src/components/Profile/Profile.jsx

```diff
@@ -14,6 +14,11 @@ class Profile extends Component {
     }
   }
 
+  componentDidMount() {
```

Comment:
Hmm I would say that whenever you're changing styles like this with javascript, it's not great style. You should generally avoid doing anything like `document.x.style.x = x` It's better to change classes.